packages/dds/tree/src/edit-manager/editManager.ts

```diff
@@ -26,18 +27,21 @@ type SeqNumber = number;
 export class EditManager<TChangeset, TChangeFamily extends ChangeFamily<any, TChangeset>> {
     private readonly trunk: Commit<TChangeset>[] = [];
     private readonly branches: Map<SessionId, Branch<TChangeset>> = new Map();
-    private readonly localChanges: TChangeset[] = [];
+    private localChanges: TChangeset[] = [];
 
     public constructor(private readonly localSessionId: SessionId,
         private readonly changeFamily: TChangeFamily,
     ) { }
 
-    public addSequencedChange(newCommit: Commit<TChangeset>, anchors?: AnchorSet): Delta.Root {
-        if (newCommit.sessionId === this.localSessionId) {
-            this.localChanges.shift();
-            return Delta.empty;
-        }
+    public getTrunk(): readonly RecursiveReadonly<Commit<TChangeset>>[] {
+        return this.trunk;
+    }
+
+    public getLocalChanges(): readonly RecursiveReadonly<TChangeset>[] {
+        return this.localChanges;
+    }
 
+    public addSequencedChange(newCommit: Commit<TChangeset>, anchors?: AnchorSet): Delta.Root {
         const branch = this.getOrCreateBranch(newCommit.sessionId, newCommit.refNumber);
```

Comment:
Sorry I delete your comment here: it highlihted the fact that my change means we now create a branch for local changes, which seems unecessary.

Reply:
Fixed